<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QDialog" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>835</width>
    <height>528</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>835</width>
    <height>528</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph BlackWood Peregrin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3d3d3d;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_x_max">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="text">
         <string>x max:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_y_min">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="text">
         <string>y min:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_x_min">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="text">
         <string>x min:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labe_formule">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="text">
         <string>Formule:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_x_max">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QDoubleSpinBox {
	border: 2px solid gray;
	border-radius: 10px;
	background-color: black;
	color: white;
	font-size: 20px;
	font-family: &quot;Comic Sans MS&quot;;
}
 QDoubleSpinBox::up-button {
	 subcontrol-origin: border;
     subcontrol-position: top right; 
     width: 16px; 
     border-width: 1px;
 }

  QDoubleSpinBox::up-arrow {
     width: 7px;
     height: 7px;
 }

  QDoubleSpinBox::down-button {
     subcontrol-origin: border;
     subcontrol-position: bottom right;
     width: 16px;
     border-width: 1px;
     border-top-width: 0;
 }

  QDoubleSpinBox::down-arrow {
     width: 7px;
     height: 7px;
 }</string>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_y_min">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QDoubleSpinBox {
	border: 2px solid gray;
	border-radius: 10px;
	background-color: black;
	color: white;
	font-size: 20px;
	font-family: &quot;Comic Sans MS&quot;;
}
 QDoubleSpinBox::up-button {
	 subcontrol-origin: border;
     subcontrol-position: top right; 
     width: 16px; 
     border-width: 1px;
 }

  QDoubleSpinBox::up-arrow {
     width: 7px;
     height: 7px;
 }

  QDoubleSpinBox::down-button {
     subcontrol-origin: border;
     subcontrol-position: bottom right;
     width: 16px;
     border-width: 1px;
     border-top-width: 0;
 }

  QDoubleSpinBox::down-arrow {
     width: 7px;
     height: 7px;
 }</string>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>-10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_y_max">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="text">
         <string>y max:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_y_max">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QDoubleSpinBox {
	border: 2px solid gray;
	border-radius: 10px;
	background-color: black;
	color: white;
	font-size: 20px;
	font-family: &quot;Comic Sans MS&quot;;
}
 QDoubleSpinBox::up-button {
	 subcontrol-origin: border;
     subcontrol-position: top right; 
     width: 16px; 
     border-width: 1px;
 }

  QDoubleSpinBox::up-arrow {
     width: 7px;
     height: 7px;
 }

  QDoubleSpinBox::down-button {
     subcontrol-origin: border;
     subcontrol-position: bottom right;
     width: 16px;
     border-width: 1px;
     border-top-width: 0;
 }

  QDoubleSpinBox::down-arrow {
     width: 7px;
     height: 7px;
 }</string>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_x_min">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QDoubleSpinBox {
	border: 2px solid gray;
	border-radius: 10px;
	background-color: black;
	color: white;
	font-size: 20px;
	font-family: &quot;Comic Sans MS&quot;;
}
 QDoubleSpinBox::up-button {
	 subcontrol-origin: border;
     subcontrol-position: top right; 
     width: 16px; 
     border-width: 1px;
 }

  QDoubleSpinBox::up-arrow {
     width: 7px;
     height: 7px;
 }

  QDoubleSpinBox::down-button {
     subcontrol-origin: border;
     subcontrol-position: bottom right;
     width: 16px;
     border-width: 1px;
     border-top-width: 0;
 }

  QDoubleSpinBox::down-arrow {
     width: 7px;
     height: 7px;
 }</string>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>-10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="but_graph">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
font: bold 20px;
color: white;
background-color: rgba(70, 12, 190, 110);
font-family: &quot;Comic Sans MS&quot;;
border-radius: 15px;
border: 3px solid rgb(12, 0, 0);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
QPushButton:hover {
	border: 4px solid gray;
	color: black;
}</string>
        </property>
        <property name="text">
         <string>Build Graph</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="textEdit_formule">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;
border: 2px solid gray;
border-radius: 10px;
background-color: black;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCustomPlot" name="widgetPlot" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;
border-radius: 10px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_complete">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>PlotLibrary/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
